boards/arm/beagleconnect_freedom_cc1352r/doc/index.rst

```diff
@@ -0,0 +1,66 @@
+.. _beagleconnect_freedom_cc1352r:
+
+BeagleBoard BeagleConnect Freedom
+#################
+
+Overview
+********
+
+The BeagleBoard BeagleConnect Freedom is a wireless
+IOT board based on the SimpleLink |trade| multi-Standard CC1352R wireless MCU.
+
+
+.. figure:: img/Beagleconnect_Freedom.jpeg
+   :width: 400px
+   :align: center
+   :alt: BeagleBoard BeagleConnect Freedom
+
+   BeagleBoard BeagleConnect Freedom |trade|
+
+Hardware
+********
+
+The BeagleBoard BeagleConnect Freedom IOT board features the CC1352R wireless MCU.
```

Comment:
No "The". Not CC1352R. Please spell out microcontroller and include MCU in parentheses. "microcontroller (MCU)". I don't know that "IOT" is desired. Please try to use language from https://github.com/jadonk/beagleconnect.